ricecooker/classes/nodes.py

```diff
@@ -526,3 +521,58 @@ def validate(self):
             return super(Exercise, self).validate()
         except AssertionError as ae:
             raise InvalidNodeException("Invalid node: {0} - {1}".format(self.title, self.__dict__))
+
+class HTML(ContentNode):
```

Comment:
Oh, shoot, I hadn't opened my PR yet, but I had written part of this already. But you've done a better job of it; I'll integrate some of my additional pieces (e.g. the "predictable zip" piece) and open as a PR against your branch.